Program.cs

```diff
@@ -15,252 +15,273 @@ public static void printsContentsRefrigerator(List<Refrigerator> allfrige)
         {
             foreach (Refrigerator friger in allfrige)
```

Comment:
triple foreach? 
not sure that its convenient.

seperate the logic more to build it more carefully.